CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ configure_file(
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc.in"
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc"
   )
-
+IF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64")
```

Comment:
Doesn't this assume only the existence of `x86` and `aarch64`? Won't this break `ppc64le` builds now?